tests/perf-test/SelectionList.perf-test.js

```diff
@@ -2,6 +2,7 @@ import {fireEvent} from '@testing-library/react-native';
 import React, {useState} from 'react';
 import {measurePerformance} from 'reassure';
 import _ from 'underscore';
+import RadioListItem from '@components/SelectionList/RadioListItem';
```

Comment:
I think the test will fail here. Only relative import paths are expected in perf test file.

Reply:
hmmm... checked the tests and everything works 🤔